client/components/data/query-sites/README.md

```diff
@@ -0,0 +1,8 @@
+Query Sites
+===========================
+
+`<QuerySites />` is a React component used in managing network requests for sites.
```

Comment:
I'm for keeping this as `<QuerySites />` with room for enhancements in the future, but agree that we might want to document its current usage more clearly (no props, requests all sites).